### ISSUE TITLE
✨Publish releases to homebrew directly

### ISSUE THREAD

**Issue by marcosnils** [dagger/dagger#2823]
### What are you trying to do?

After dagger releases are crafted, homebrew update should fetch the latest version. There's a GH action that can be used for this. 

ref: https://github.com/Homebrew/homebrew-core/pull/105746#issuecomment-1190567751

cc @gerhard 

### Why is this important to you?

keep dagger updated

### How are you currently working around this?

downloading the new binary without homebrew.

**Comment by gerhard** [comment]
Yes, that's right, our tap gets updated correctly.

This issue is for the `dagger` formula which is part of the `homebrew-core`, meaning that no tap is required. If we get this working properly, then maybe we don't even need to have our own brew tap... I'm all for simplifying this aspect. WDYT @aluzzardi?

**Comment by jpadams** [comment]
I'd love to see this. It usually takes several days for the latest version of `dagger` to show up in Homebrew today. It used to be quicker, but now that I see humans are in the loop, it's bound to be variable and laggy. https://github.com/Homebrew/homebrew-core/pull/105746

**Comment by gerhard** [comment]
While I ran out of time this week, I intend to come back to this next week.

**Comment by gerhard** [comment]
I intended to get to this today, but ran out of time. If we can keep the release paused for an extra day @marcosnils @aluzzardi, then I will get to this tomorrow, push the release through manually, and make sure that everything works as expected with this new addition. If we want, we should be good to leave the Auto-Release enabled so that it gets triggered automatically next week.

**Comment by gerhard** [comment]
TL;DR this is now handled part of the release process, the PRs above have all the context.

We tested it via https://github.com/Homebrew/homebrew-core/pull/107233. This is now already available in Homebrew:

<img width="970" alt="image" src="https://user-images.githubusercontent.com/3342/182668566-fce08e27-423c-4831-85c8-8f5dcc4cf06c.png">

One thing which I have noticed is that the bottled formula is significantly different to the tap one:
- https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/dagger.rb
- https://github.com/dagger/homebrew-tap/blob/main/dagger.rb

While this is OK for now, at some point in the future we should converge the two.
